framework/console/controllers/MessageController.php

```diff
@@ -78,22 +245,25 @@ public function actionConfig($filePath)
      * this file and then customize it for your needs.
      * @throws Exception on failure.
      */
-    public function actionExtract($configFile)
+    public function actionExtract($configFile = null)
```

Comment:
Also we need to update to the PHPDoc and describe behavior in case when `$configFile` is null